com.jetbrains.Rider.yaml

```diff
@@ -0,0 +1,55 @@
+
+app-id: com.jetbrains.Rider
+runtime: org.freedesktop.Sdk
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: rider-desktop
+separate-locales: false
+tags:
+  - proprietary
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --socket=session-bus
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Flatpak
+  - --device=all
+  - --env=RIDER_JDK=${FLATPAK_DEST}/extra/rider/jre64
+modules:
+  - name: rider
+    buildsystem: simple
+    build-commands:
+      - unzip rider_logos.zip
+      - install -D apply_extra /app/bin/apply_extra
+      - install -D -m755 rider.sh ${FLATPAK_DEST}/bin/rider-desktop
+      - install -D -m644 logo.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - install -D -m644 -t ${FLATPAK_DEST}/share/applications/ ${FLATPAK_ID}.desktop
+      - install -D -m644 -t ${FLATPAK_DEST}/share/metainfo/ ${FLATPAK_ID}.appdata.xml
+    sources:
+      - type: extra-data
+        url: https://download.jetbrains.com/rider/JetBrains.Rider-2019.3.4.tar.gz
+        sha256: e2a0c419c235529320f0c043155c85bc86032ac61717191593f5b63036d85d9d
+        filename: rider.tar.gz
+        x-checker-data:
+          type: jetbrains
+          code: RD
+        size: 661849418
+      - type: file
+        path: com.jetbrains.Rider.desktop
+      - type: file
+        path: com.jetbrains.Rider.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html